formbuilder/config.js

```diff
@@ -194,6 +207,65 @@ export default {
         },
       },
       formData: {}
+    },
+    {
+      id: "address",
+      icon: "leaf",
+      label: "Postal address",
+      jsonSchema: {
+        type: "object",
+        title: "",
+        description: "",
+        properties: {
+          address1: {type: "string", title: "Address line 1"},
+          address2: {type: "string", title: "Address line 2"},
+          zip: {type: "string", title: "Zip code"},
+          city: {type: "string", title: "City"},
+          state: {type: "string", title: "State"},
+          country: {type: "string", title: "Country", enum: countries},
+        }
+      },
+      uiSchema: {
+        editSchema: {
+          type: "object",
+          properties: {
+            name: {type: "string", title: "Field name"},
+            title: {type: "string", title: "Label"},
+            description: {type: "string", title: "Description"},
+            required: {type: "boolean"},
+          }
+        },
+      },
+      formData: {}
     }
+  ],
+  fieldSets: [
+    // {
```

Comment:
WiP as stated in the description ;) The goal is to have a separate section listing predefined fieldsets.